keel-core/src/main/kotlin/com/netflix/spinnaker/keel/actuation/ResourceActuator.kt

```diff
@@ -120,12 +132,6 @@ class ResourceActuator(
 
               if (versionedArtifact != null) {
                 with(versionedArtifact) {
-                  val deliveryConfig = deliveryConfigRepository.deliveryConfigFor(resource.id)
-                  val environment = deliveryConfig.environmentFor(resource)?.name
-                    ?: error(
-                      "Failed to find environment for ${resource.id} in deliveryConfig ${deliveryConfig.name} " +
-                        "while attempting to veto artifact $artifactType:$artifactName version $artifactVersion"
-                    )
```

Comment:
Just pulled this up earlier in the method since I needed to get the delivery config